code/game/machinery/vendors/contraband_vendors.dm

```diff
@@ -179,3 +179,20 @@
 
 	armor = list(MELEE = 100, BULLET = 100, LASER = 100, ENERGY = 100, BOMB = 0, RAD = 0, FIRE = 100, ACID = 50)
 	resistance_flags = FIRE_PROOF
+
+/obj/machinery/economy/vending/toyliberationstation/crowbar_act(mob/user, obj/item/I)
```

Comment:
Two things:

First, I'm not sure you need to specify all this behavior here if you're just going to prevent it from working anyway. If I were you, I'd just make an override that has the to_chat() and nothing else.

Second, I feel like it might still be nice to have the ability to un- and re-wrench toy liberation stations that are found elsewhere (such as from admin events). I think it'd be better to make a new syndicate version of this vendor, much like we do with other things found on the base (like the nanomed plus), and implement the changes I suggested above on its overrides.

Reply:
A new type of vendor moveabletoyliberationstation has been added to allow admins to place a moveable one.